### PR TITLE
Add glossary to site

### PR DESCRIPTION
**Pull request by kevinmpowell**
I think I did this correctly. Lmk if I need to make any changes.

I'll be OOO tomorrow, but back Monday, June 14.

### CODE REVIEW

**Review by mattfelten — APPROVED** [review]
LGTM on the code end

**Review by jina — APPROVED** [review]

File: pages/glossary.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+The single source of truth to name and store a design decision, distributed so teams can use it across design tools and coding languages.
+
+## Design token translation tool
+
+A tool that translates token sets from one format (such as YAML) to a range of other formats (such as CSS custom properties, Sass, Swift, Sketch palettes…).
+
+### Examples
+
+Theo
```
Comment (kaelig) [inline pages/glossary.md:23]: Is this meant to be a list instead

File: pages/glossary.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+## Variable
+
+Generic term describing the most common way (but not the only way) a design token is formatted and used in code.
+
+### Examples
+
+CSS custom property
+Sass variable
+Javascript variable
```
Comment (kaelig) [inline pages/glossary.md:56]:
```suggestion
JavaScript variable
```

File: pages/glossary.md (hunk 1 above)
Comment (jina) [reply pages/glossary.md:23]: ah yeah. good point. lots of things in this draft can be bulleted lists! :)

Comment (kevinmpowell) [reply pages/glossary.md:23]: Should I use a definition list?
`<dl>`

Comment (c1rrus) [reply pages/glossary.md:23]: `<dl>` would be the most semantically approriate elemeny for this IMHO.

Note you can have multiple `<dt>`s per `<dd>`, which might be useful to terms that have synonyms.

Comment (kaelig) [reply pages/glossary.md:23]: I agree, regular (Markdown) unordered lists should do!

File: pages/glossary.md (hunk 2 above)
Comment (kevinmpowell) [reply pages/glossary.md]: Applied

File: pages/glossary.md (hunk 1 above)
Comment (kevinmpowell) [reply pages/glossary.md]: I've updated the PR and added the unordered list syntax. I kept the h2's and h3's for visual organization. I made the h3's nested list items.

Comment (kaelig) [reply pages/glossary.md]: Oh I see, I think there was a miscommunication somewhere.

I'm not sure we want each and every single definition title and "Examples" to be list items?

https://60ca4d3c631f9c00078002bb--designtokensorg.netlify.app/glossary/

Also: can you style the unordered lists so they have bullet points?

File: pages/glossary.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- ## Design token translation tool
+
+  A tool that translates token sets from one format (such as YAML) to a range of other formats (such as CSS custom properties, Sass, Swift, Sketch palettes…).
+
+  - ### Examples
+    Theo
+    Style Dictionary
+    Diez
+    Specify
+
```
Comment (kaelig) [inline pages/glossary.md:26]: Here's what I had in mind re:lists, what do you think?

```suggestion
## Design token

The single source of truth to name and store a design decision, distributed so teams can use it across design tools and coding languages.

## Design token translation tool

A tool that translates token sets from one format (such as YAML) to a range of other formats (such as CSS custom properties, Sass, Swift, Sketch palettes…).

### Examples

- Theo
- Style Dictionary
- Diez
- Specify

```

Comment (kevinmpowell) [reply pages/glossary.md:26]: I'll take a closer look later this week.

**Comment by kevinmpowell** [comment]
@kaelig I think this is in good shape now

**Review by kaelig — APPROVED** [review]

**Pull request merged**